File: _persist/temp/venda/CSharp/Histrel.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("HISTREL")]
    public class Histrel : CollectionItem
    {
        [Campo("ID_HISTREL", CampoTipo.tfKey)]
        public int Id_Histrel { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("TP_DOCUMENTO", CampoTipo.tfReq)]
        public int Tp_Documento { get; set; }
        [Campo("CD_HISTREL", CampoTipo.tfReq)]
        public int Cd_Histrel { get; set; }
        [Campo("DS_HISTREL", CampoTipo.tfReq)]
        public string Ds_Histrel { get; set; }
        [Campo("NR_PARCELAS", CampoTipo.tfReq)]
        public int Nr_Parcelas { get; set; }
    }
}

File: _persist/temp/venda/Java/Histrel.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("HISTREL")
public class Histrel extends CollectionItem
{
    @Campo("ID_HISTREL", CampoTipo.tfKey)
    public Integer Id_Histrel;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("TP_DOCUMENTO", CampoTipo.tfReq)
    public Integer Tp_Documento;
    @Campo("CD_HISTREL", CampoTipo.tfReq)
    public Integer Cd_Histrel;
    @Campo("DS_HISTREL", CampoTipo.tfReq)
    public String Ds_Histrel;
    @Campo("NR_PARCELAS", CampoTipo.tfReq)
    public Integer Nr_Parcelas;
}